sample/src/main/java/com/clevertap/demo/ui/main/HomeScreenFragment.kt

```diff
@@ -220,4 +226,13 @@ class HomeScreenFragment : Fragment() {
             }
         }
     }
+
+    private fun applySystemBarsInsets(view: ExpandableListView) {
+        view.clipToPadding = false
+        ViewCompat.setOnApplyWindowInsetsListener(view) { v: View, insets: WindowInsetsCompat ->
+            val bars: Insets = insets.getInsets(WindowInsetsCompat.Type.systemBars())
+            v.setPadding(bars.left, 0, bars.right, bars.bottom)
+            WindowInsetsCompat.CONSUMED
+        }
+    }
```

Comment:
paddings to list and clip to padding are used, other places set insets as view margins